dd-sdk-android-compose/src/main/kotlin/com/datadog/android/compose/InteractionTracking.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.compose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.State
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberUpdatedState
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.RumActionType
+import com.datadog.android.rum.RumAttributes
+import com.datadog.android.rum.RumMonitor
+
+/**
+ * Creates a proxy around click listener, which will report clicks to Datadog.
+ *
+ * @param targetName Name of the click target.
+ * @param attributes additional custom attributes to attach to the action. Attributes can be
+ * nested up to 9 levels deep. Keys using more than 9 levels will be sanitized by SDK.
+ * @param onClick Click listener.
+ */
+@ExperimentalTrackingApi
+@Composable
+fun trackClicks(
```

Comment:
Shouldn't it be "taps", not "clicks" 🤔💭?

Reply:
good question! For now it is `click`, because this comes as a wrapper of `onClick` attribute which has specific type. I need to explore other interactions names and types in Compose framework to see if I can generalize this wrapper better.

Reply:
I think I will stick with `click` terminology to be consistent with `onClick` and `onLongClick` arguments in Compose for now, and use `tap` terminology internally to be consistent with `RumActionType.TAP`.